shell/assets/translations/en-us.yaml

```diff
@@ -2940,7 +2940,6 @@ istio:
   policy: Enable Policy
   pilotRequired: Pilot must be enabled in order to enable Kiali
   poweredBy: Powered by <a target="_blank" rel="noopener noreferrer nofollow" href='https://istio.io/latest/'>Istio</a>
-  telemetry: Enable Telemetry
```

Comment:
This is in the istio block, looking at the code it relates to the istio chart and a telemetry setting there. Are you sure it relates to rancher telemetry?

Reply:
Good catch, I shouldn't have removed this. Fixing it now, as well as it's counterpart in shell/assets/translations/zh-hans.yaml.